easybuild/easyblocks/t/trinity.py

```diff
@@ -319,11 +319,16 @@ def sanity_check_step(self):
         else:
             sep = '_r'
 
+        if version >= LooseVersion('2.8'):
```

Comment:
@pescobar After a quick test, I noticed that this breaks the installation of some old `Trinity` easyconfigs like `Trinity-2012-10-05-goolf-1.4.10.eb`:

```
eb Trinity-2012-10-05-goolf-1.4.10.eb -df --include-easyblocks trinity.py
...
== sanity checking...
  >> file 'trinityrnaseq_r2012-10-05/Inchworm/bin/inchworm' found: OK
  >> file 'trinityrnaseq_r2012-10-05/Chrysalis/bin/Chrysalis' found: FAILED
```

Maybe enhance this to:

```suggestion
if version >= LooseVersion('2.8') and version < LooseVersion('2000'):